SimplenoteUITests/EmailLogin.swift

```diff
@@ -14,7 +14,18 @@ class EmailLogin {
         backButton.tap()
     }
 
-    class func logIn(email: String, password: String) {
+    class func logIn() {
+        var testAccount = ProcessInfo.processInfo.environment["UI_TEST_ACCOUNT"]!
+
+        // Use 'default' account if test account was not passed via environment variable
+        if testAccount.isEmpty {
+            testAccount = testDataEmail
+        }
+
+        EmailLogin.logIn(testAccount, testDataPassword)
+    }
+
+    class func logIn(_ email: String, _ password: String) {
```

Comment:
Nitpick:

```suggestion
    class func logIn(email: String, password: String) {
```

The [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/#weak-type-information) recommend to precede each weakly typed parameter, such as `Any`, `Int`, or `String, with a noun describing its role:

<img width="500" alt="screen shot 2019-02-06 at 1 43 58 pm" src="https://user-images.githubusercontent.com/1218433/52317755-78bf6600-2a15-11e9-93ba-ec4a22fb71cb.png">

It's true that Xcode's code completion would suggest that the first argument should be email and the second password, but I find having them explicit helps with readability.

Reply:
This is nice to know, thanks! I previously thought that using nameless parameters is something dictated by gut feeling rather than having a more well-define rule, like not using it for weakly typed parameters.